plugins/tekton/src/hooks/useTektonViewPermission.ts

```diff
@@ -0,0 +1,19 @@
+import { catalogEntityReadPermission } from '@backstage/plugin-catalog-common/alpha';
+import { usePermission } from '@backstage/plugin-permission-react';
+
+import { tektonViewPermission } from '@janus-idp/backstage-plugin-tekton-common';
+
+export const useTektonViewPermission = () => {
+  const tektonViewPermissionResult = usePermission({
+    permission: tektonViewPermission,
+  });
+
+  const catalogEntityPermissionResult = usePermission({
+    permission: catalogEntityReadPermission,
+    resourceRef: catalogEntityReadPermission.resourceType,
+  });
```

Comment:
```suggestion
```

Reply:
@PatAKnight Thanks for the heads up, I have updated both the PRs.